### PR TITLE
FS2-2856: fix plot borders

### PR DESCRIPTION
**Pull request by chschan**
This PR fixes
1) The extra space added between the quadrant rectangles and the axes; and
2) The plot border not closing up properly when we drew it manually with lines (also has the nice side-effect that we can remove some code, now that I realise what `mirror` does)
Screenshot 2024-08-28 at 1 58 40 AM

### CODE REVIEW

**Comment by wooneh** [comment]
I found that double-clicking on the chart "resets" the chart so that there is still a gap. Is there a way to fix it? I guess in some way the double-click resets the autorange

**Comment by chschan** [comment]
Thanks for picking that up. I think I've fixed the gap after double-clicking in [524b3bf](https://github.com/Displayr/rhtmlCombinedScatter/pull/81/commits/524b3bf1fca454dc54bc779502e6fe5589d5cb23). It kind of blinks because I couldn't figure out how to suppress what it normally does with a double click. But it fixes itself up afterwards.

File: theSrc/scripts/LabeledScatter.js [hunk 1]
```diff
@@ -176,20 +177,34 @@ class LabeledScatter {
           await FitLine.draw(this.rootElement, config)
         }
         if (config.quadrantsShow) {
+          const original_x_range = _.cloneDeep(plotlyChart._fullLayout.xaxis.range)
```
Comment (wooneh) [inline theSrc/scripts/LabeledScatter.js:180]: Does it need a deep copy, isn't this just an array? i.e.
```suggestion
          const original_x_range = [...plotlyChart._fullLayout.xaxis.range]
```

**Review by wooneh — APPROVED** [review]

File: theSrc/scripts/LabeledScatter.js (hunk 1 above)
Comment (chschan) [reply theSrc/scripts/LabeledScatter.js:180]: No, when I try it with a shallow copy the the array updates with the new values after I zoom. It needs the deep copy to keep the original values after the initial draw.

**Pull request merged**